common/api/components-react.api.md

```diff
@@ -2406,20 +2401,10 @@ export abstract class PropertyRecordDataFiltererBase extends PropertyDataFiltere
 }
 
 // @public
-export class PropertyRenderer extends React_3.Component<PropertyRendererProps, PropertyRendererState> {
-    constructor(props: PropertyRendererProps);
-    // (undocumented)
-    componentDidMount(): void;
-    // (undocumented)
-    componentDidUpdate(prevProps: PropertyRendererProps): void;
-    // (undocumented)
-    static getLabelOffset(indentation?: number, orientation?: Orientation, width?: number, columnRatio?: number, minColumnLabelWidth?: number): number;
-    // (undocumented)
-    render(): React_3.JSX.Element;
-    // (undocumented)
-    readonly state: Readonly<PropertyRendererState>;
-    updateDisplayValueAsEditor(props: PropertyRendererProps): void;
```

Comment:
Do we need `useImperativeHandle` here or are we hoping this is not used?

Reply:
I hope this is not used. I could not find references to it in AppUI repo and I hope it is not used by any consumer. I even think `PropertyRenderer` is not used directly. Only inside PropertyGrid.